src/components/PokemonModalWindow/PokemonModalWindow.js

```diff
@@ -1,63 +1,90 @@
-                    <CloseIcon />
-                </IconButton>
-            ) : null}
-        </MuiDialogTitle>
-    );
+  const classes = useDialogTitleStyles();
+  const { children, onClose } = props;
+  return (
+    <MuiDialogTitle disableTypography {...props} className={classes.root}>
+      <Typography variant="h6">{children}</Typography>
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          edge="start"
+          className={classes.closeButton}
+          onClick={onClose}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </MuiDialogTitle>
+  );
 };
 
 export default function PokemonModalWindow() {
+  const classes = useDialogStyle();
+  const dispatch = useDispatch();
+  const {
+    selectedPokemon: pokemonObject,
+    showPokemonModalWindow: showModalWindow,
+  } = useSelector((state) => state.selectedPokemon);
 
-    const classes = useDialogStyle();
-    const dispatch = useDispatch();
-    const {selectedPokemon: pokemonObject, showPokemonModalWindow: showModalWindow} = useSelector(state => state.selectedPokemon);
-
-    const handleToggleModalWindow = () => {
-        dispatch({type: actionTypes.TOGGLE_POKEMON_MODAL_WINDOW})
-    }
+  const handleToggleModalWindow = () => {
+    dispatch({ type: actionTypes.TOGGLE_POKEMON_MODAL_WINDOW });
+  };
 
-    return (
-        <div>
-            <Dialog onClose={handleToggleModalWindow} aria-labelledby="customized-dialog-title" open={showModalWindow} maxWidth="md">
-                <DialogTitle id="customized-dialog-title" onClose={handleToggleModalWindow}>
-                    {`# ${pokemonObject.id} ${pokemonObject.name}`}
-                </DialogTitle>
-                <DialogContent dividers className={classes.dialogContent}>
-                    <img className={classes.pokemonSprite} src={pokemonObject.sprite} alt='pokemon-sprite'/>
-                    <DialogContent dividers className={classes.dialogContentDescription}>
-                        <Typography gutterBottom>
-                             {`Height: ${pokemonObject.height}`}
-                        </Typography>
-                        <Typography gutterBottom>
-                            {`Weight: ${pokemonObject.weight}`}
-                        </Typography>
-                    </DialogContent>
-                </DialogContent>
-            </Dialog>
-        </div>
-    );
+  return (
+    <div>
+      <Dialog
+        onClose={handleToggleModalWindow}
+        aria-labelledby="customized-dialog-title"
+        open={showModalWindow}
+        maxWidth="md"
+      >
+        <DialogTitle
+          id="customized-dialog-title"
+          onClose={handleToggleModalWindow}
+        >
+          {`# ${pokemonObject.id} ${pokemonObject.name}`}
+        </DialogTitle>
+        <DialogContent dividers className={classes.dialogContent}>
+          <img
```

Comment:
It's better to wrap images in an extra block to make sure image' styles are correct and the image will display on the page correctly.

Reply:
Done